semantic-codeql/src/Language/CodeQL/Tags.hs

```diff
@@ -100,19 +100,22 @@ instance ToTags CodeQL.DatatypeBranch where
     , name = CodeQL.ClassName { text }
     } = yieldTag text Class loc byteRange >> gtags t
 
-instance ToTags CodeQL.HigherOrderTerm where
-  tags t@CodeQL.HigherOrderTerm
-    { ann = loc@Loc { byteRange }
-    , name = CodeQL.LiteralId { text }
-    } = yieldTag text Call loc byteRange >> gtags t
-
```

Comment:
@alexet noted this `ToTags` instance is unnecessary as it always resolves to special built-in QL literals.